libraries/AP_Camera/AP_Camera_Scripting.cpp

```diff
@@ -16,7 +16,11 @@ bool AP_Camera_Scripting::trigger_pic()
 // set start_recording = true to start record, false to stop recording
 bool AP_Camera_Scripting::record_video(bool start_recording)
 {
+    if ((start_recording & is_recording) == 1U) {
```

Comment:
```suggestion
    if (start_recording && is_recording) {
```
Although again I don't think you should block.